qemu/tests/block_discard_hotplug.py

```diff
@@ -0,0 +1,114 @@
+"""
+hot-plug discard disk testing
+"""
+import logging
+
+from avocado.utils import process
+
+from virttest import storage
+from virttest import error_context
+from provider.block_devices_plug import BlockDevicesPlug
+from virttest import data_dir
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Qemu discard hotplug support test:
+
+    1) Boot vm
+    2) Hot-plug the data disk with discard option
+    3) Format the data disk and mount it in guest
+    4) Execute dd command to the mounted disk in guest
+    5) Check disk size allocation
+    6) Execute rm and fstrim command in guest
+    7) Check disk size allocation
+    8) Hot-unplug the data disk
+    9) Reboot vm
+
+
+    :param test: kvm test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+
+    def get_scsi_debug_disk(guest_session=None):
+        """"
+        Get scsi debug disk on host or on guest which created as scsi-block.
+        """
+        cmd = "lsblk -S -n -p|grep scsi_debug"
+
+        if guest_session:
+            status, output = guest_session.cmd_status_output(cmd)
+        else:
+            status, output = process.getstatusoutput(cmd)
```

Comment:
```suggestion
        status, output = process.getstatusoutput(cmd)
```
Better to remove the part of guest session then modify the docstring `Get scsi debug disk on host which created as scsi-block.`

Reply:
updated